spec/build-confirm-spec.js

```diff
@@ -4,14 +4,19 @@ import _ from 'lodash';
 import fs from 'fs-extra';
 import temp from 'temp';
 import specHelpers from 'atom-build-spec-helpers';
+import os from 'os';
+
```

Comment:
It's just that row 8 is unnecessary and there are now two blank lines between the import statement and the next statement: http://eslint.org/docs/rules/no-multiple-empty-lines

The ESlint configs for atom-build [requires this](https://github.com/AtomBuild/eslint-config-atom-build/blob/master/stylistic-issues.js#L64), but that was added in a later version of that config. I will upgrade, but until then, feel free to fix or ignore this comment as you please.